### PR TITLE
doc: add information about firewall in the docs

### LINKED ISSUE #648: configurable number of max dbus/connections messages node controller can handle from each agent node
## Describe the bug

Scenario:
Imagine having 3 nodes or more and each node sending 2M dbus messages at the same time to bluechi-controller.
We might have bluechi-controller overloaded with a big queue to handle. 

Possible ideas for investigation:
- Create a document explaining via firewalld how to limit max connections per node per hour/minute.

- Implement a configurable (controller.conf) mechanism that could set the maximum number of messages per node (per hour/minute). By default this option should be disable but users can set their own limits if they want.

In repo bluechi has a flood tool available to help create such scenario:
https://github.com/eclipse-bluechi/bluechi/tree/main/tests/tools/FFI

### PR DESCRIPTION
**Pull request by dougsland**
Add documentation to users regarding firewalld and iptables for BlueChi.

Fixes: https://github.com/eclipse-bluechi/bluechi/issues/648

### CODE REVIEW

File: doc/docs/network/index.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+sudo iptables -A INPUT -p tcp --dport 842 -m conntrack --ctstate NEW -m recent --name BLUECHIRULE --set
+
+sudo iptables -A INPUT -p tcp --dport 842 -m conntrack --ctstate NEW -m recent --name BLUECHIRULE --update --seconds 60 --hitcount 5 -j DROP
+```
+
+## Securing connections
+
+Briefly mentioning that the communication between the BlueChi components is, by default, not secured and that the getting started guide provides an possible approach using a double proxy to achieve that.
+
+For more information, [see our documentation page](https://bluechi.readthedocs.io/en/latest/getting_started/securing_multi_node/).
```
Comment (engelmi) [inline doc/docs/network/index.md]: NIT: Please change to a relative path. Then it is clear that its an internal page. 
```suggestion
For more information, [see our documentation page](../getting_started/securing_multi_node.md).
```

**Review by engelmi — APPROVED** [review]
One small NIT, otherwise LGTM

**Comment by dougsland** [comment]
All requests addressed.

**Comment by engelmi** [comment]
> All requests addressed.

Yes, lets merge. The testing farm issues are unrelated and are, hopefully, resolved soon.

**Pull request merged**